cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/versioning/VersionValidator.java

```diff
@@ -69,6 +78,18 @@ public VersionValidator(Type validatorType, String cobiGenVersion) {
    */
   public void validate(float configVersion) {
 
+    validate(configVersion, true);
```

Comment:
Why do we use true for the default value?

Reply:
Needs to be checked.

Reply:
added an issue + TODO now.